main.go

```diff
@@ -28,31 +28,34 @@ import (
 )
 
 type config struct { // nolint
-	awsRegion        string // AWS_REGION
-	awsAPIEndpoint   string // AWS_API_ENDPOINT
-	s3Bucket         string // AWS_S3_BUCKET
-	s3KeyPrefix      string // AWS_S3_KEY_PREFIX
-	indexDocument    string // INDEX_DOCUMENT
-	directoryListing bool   // DIRECTORY_LISTINGS
-	dirListingFormat string // DIRECTORY_LISTINGS_FORMAT
-	httpCacheControl string // HTTP_CACHE_CONTROL (max-age=86400, no-cache ...)
-	httpExpires      string // HTTP_EXPIRES (Thu, 01 Dec 1994 16:00:00 GMT ...)
-	basicAuthUser    string // BASIC_AUTH_USER
-	basicAuthPass    string // BASIC_AUTH_PASS
-	port             string // APP_PORT
-	host             string // APP_HOST
-	accessLog        bool   // ACCESS_LOG
-	sslCert          string // SSL_CERT_PATH
-	sslKey           string // SSL_KEY_PATH
-	stripPath        string // STRIP_PATH
-	contentEncoding  bool   // CONTENT_ENCODING
-	corsAllowOrigin  string // CORS_ALLOW_ORIGIN
-	corsAllowMethods string // CORS_ALLOW_METHODS
-	corsAllowHeaders string // CORS_ALLOW_HEADERS
-	corsMaxAge       int64  // CORS_MAX_AGE
-	healthCheckPath  string // HEALTHCHECK_PATH
-	allPagesInDir    bool   // GET_ALL_PAGES_IN_DIR
-	insecureTLS      bool   // Disables TLS validation on request endpoints.
+	awsRegion          string        // AWS_REGION
+	awsAPIEndpoint     string        // AWS_API_ENDPOINT
+	s3Bucket           string        // AWS_S3_BUCKET
+	s3KeyPrefix        string        // AWS_S3_KEY_PREFIX
+	indexDocument      string        // INDEX_DOCUMENT
+	directoryListing   bool          // DIRECTORY_LISTINGS
+	dirListingFormat   string        // DIRECTORY_LISTINGS_FORMAT
+	httpCacheControl   string        // HTTP_CACHE_CONTROL (max-age=86400, no-cache ...)
+	httpExpires        string        // HTTP_EXPIRES (Thu, 01 Dec 1994 16:00:00 GMT ...)
+	basicAuthUser      string        // BASIC_AUTH_USER
+	basicAuthPass      string        // BASIC_AUTH_PASS
+	port               string        // APP_PORT
+	host               string        // APP_HOST
+	accessLog          bool          // ACCESS_LOG
+	sslCert            string        // SSL_CERT_PATH
+	sslKey             string        // SSL_KEY_PATH
+	stripPath          string        // STRIP_PATH
+	contentEncoding    bool          // CONTENT_ENCODING
+	corsAllowOrigin    string        // CORS_ALLOW_ORIGIN
+	corsAllowMethods   string        // CORS_ALLOW_METHODS
+	corsAllowHeaders   string        // CORS_ALLOW_HEADERS
+	corsMaxAge         int64         // CORS_MAX_AGE
+	healthCheckPath    string        // HEALTHCHECK_PATH
+	allPagesInDir      bool          // GET_ALL_PAGES_IN_DIR
+	maxIdleConns       int           // MAX_IDLE_CONNECTIONS
+	idleConnTimeout    time.Duration // IDLE_CONNECTION_TIMEOUT
+	disableCompression bool          // DISABLE_COMPRESSION
+	insecureTLS     　 bool   // Disables TLS validation on request endpoints.
```

Comment:
__[Golint]__ illegal character U+3000 (and 1 more errors) ([see on Sider](https://sider.review/gh/repos/51703546/pulls/29#issue-4798429))

<sub>posted by Sider</sub>